src/oatpp/core/async/worker/IOEventWorker_epoll.cpp

```diff
@@ -162,8 +162,8 @@ void IOEventWorker::waitEvents() {
     OATPP_LOGE("[oatpp::async::worker::IOEventWorker::waitEvents()]", "Error:\n"
                "errno=%d\n"
                "in-events=%d\n"
-               "foreman=%d\n"
-               "this=%d\n"
+               "foreman=%p\n"
```

Comment:
Please use `%llu` or `%llx` for pointers with cast to `v_ubuff_size`.
To have unified output on all platforms

Reply:
done